src/Utils.vala

```diff
@@ -740,6 +740,17 @@ public class Utils : GLib.Object {
                 popover_background = "#15151B";
                 row_selected_color = "shade (#ffffff, 0.125)";
 
+                Gtk.Settings.get_default ().gtk_application_prefer_dark_theme = true;
+            } else if (appearance_mode == 3) {
+                base_color = "#353945";
+                check_border_color = "grey";
+                projectview_color = "#404552";
+                pane_color = "#353945";
+                pane_selected_color = "#2B303B";
+                pane_text_color = "#fefeff";
+                popover_background = "#353945";
+                row_selected_color = "shade (#fefeff, 0.25)";
```

Comment:
I think we should improve the border color of the task, maybe a deeper black.
![image](https://user-images.githubusercontent.com/33765137/81477728-e04aff00-91de-11ea-8160-148cc036cb64.png)

Reply:
Just pushed an update, do you like the new one better?